ansible/roles/dev-desktop/files/scripts/link_rust.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# Enable strict mode for Bash
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -euo pipefail
+IFS=$'\n\t'
+
+# Discover target triple (e.g. "aarch64-unknown-linux-gnu")
+target="$(rustc -vV | awk '/host/ { print $2 }')"
+
+for D in rust*; do
+  if [ -d "${D}" ]; then
+    pushd "${D}"
+
+    ./x.py build
```

Comment:
I don't know enough about developing Rust to have an opinion on this. The previous version of the script failed since it couldn't find `stage1` or `stage2`, so I put the build command in there. With the `if` statement that follows the build command, we shouldn't get errors anymore even if Rust hasn't been built yet. So I'm fine removing the build command here.

Reply:
If we remove the build command, then this script won't do anything at all. My preference is the first thing I said, avoid x.py altogether and `touch` the relevant files so we can link a rustup toolchain without waiting for rustc to compiler.

Reply:
My assumption is that these scripts are not designed with onboarding in mind, but instead to help folks when they're already working on the dev desktop. While the script doesn't to anything on a fresh installation, it is probably useful when you've manually compiled stages 1 or 2 in your own fork?